src/app/components/layout/Tooltip.scss

```diff
@@ -0,0 +1,45 @@
+@import '../../../scss/fonts';
+
+.tooltip {
+    position: relative;
+    display: flex;
+
+    .wrap {
+        visibility: hidden;
+        position: absolute;
+        top: 100%;
+        margin-top: 10px;
+        margin-left: 20px;
+        padding: 12px;
+        border-radius: 0.25rem;
+        background: #485776;
```

Comment:
Add variable to theme/dark.scss and theme/light.scss and use that variable instead of hard-coded color hex.
Add colors that make sens for light and dark themes.

Reply:
Done.